orchestra/communication/staffing.py

```diff
@@ -148,3 +151,40 @@ def send_request_inquiries(staffbot, request, worker_batch_size):
              .format(request.task)))
         request.status = StaffBotRequest.Status.COMPLETE.value
         request.save()
+
+
+def get_available_requests(worker):
+    # We want to show a worker only requests for which there is no
+    # winner or for which they have not already replied.
+    won_responses = StaffingResponse.objects.filter(is_winner=True)
```

Comment:
pay particular attention to the logic here---i think i cover it with tests, but it wouldn't hurt to think carefully until line 172 :)